Encounter Me.Api/Controllers/CapturePointController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Encounter_Me.Api.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Encounter_Me.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CapturePointController : Controller
+    {
+        private readonly ICapturePointRepository _capturePointRepository;
+
+        public CapturePointController(ICapturePointRepository capturePointRepository)
+        {
+            _capturePointRepository = capturePointRepository;
+        }
+
+        [HttpGet]
+        public IActionResult GetAllTokens()
+        {
+            return Ok("Here you go."); // For testing only
+        }
```

Comment:
What a method from Token Controller is doing in here?

Reply:
was used for testing, removed in newest commit